tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/cope_survey_response_suppression_test.py

```diff
@@ -0,0 +1,146 @@
+"""
```

Comment:
All the cleaning rules that are being applied on controlled_tier are located under cleaning_rules/deid directory, if possible please move this test to the appropriate directory other than that everything looks fine.

Reply:
Okie dokie! I will move these modules!